File: CompilationTest/ExplicitGenericTypeTest/results/ExplicitGenericTypeTest.py
class TestClass:
    def test_method(self):
        result = ["y"]
        map = {
          "x": 5,
        }
        keys = map.keys()
        print result[0]
        print keys[0]

try:
    TestClass().test_method()
except Exception as err:
    print "Exception: " + err.message

File: CompilationTest/ExplicitGenericTypeTest/results/ExplicitGenericTypeTest.php
<?php

class TestClass {
    function testMethod() {
        $result = array("y");
        $map = array(
          "x" => 5,
        );
        $keys = array_keys($map);
        print($result[0] . "\n");
        print($keys[0] . "\n");
    }
}

try {
    $c = new TestClass();
    $c->testMethod();
} catch (Exception $err) {
    echo 'Exception: ' . $err->getMessage() . "\n";
}